### PR TITLE
PSP: func_80171ED4

### PR DESCRIPTION
**Pull request by sonicdcer**

### CODE REVIEW

File: src/servant/tt_000/10E8.c [hunk 1]
```diff
@@ -468,86 +468,82 @@ void func_801719E0(Entity* self) {
...
+    *spriteBanks = (SpriteParts*)D_80170040;
 
     e = &g_Entities[4];
+
     DestroyEntity(e);
+
     e->unk5A = 0x6C;
     e->palette = 0x140;
     e->animSet = ANIMSET_OVL(20);
+    e->zPriority = g_Entities[PLAYER_CHARACTER].zPriority - 2;
```
Comment (Xeeynamo) [inline src/servant/tt_000/10E8.c:520]: shall we change these back to `PLAYER`?

Comment (sonicdcer) [reply src/servant/tt_000/10E8.c]: Done

**Comment by Xeeynamo** [comment]
The CI is finally green! The needed changes were adding the BSS section in the splat config, solve a few fake symbols and match `func_80171ED4`. Thanks to @Mc-muffin and @1superchip for the `func_80173C2C` match!

@sonicdcer please give my changes a second look in case I missed something

File: src/servant/tt_000/10E8.c [hunk 2]
```diff
@@ -468,86 +463,80 @@ void func_801719E0(Entity* self) {
...
+
+    for (i = 0; i < 256; i++) {
         *dst++ = *src++;
     }
 
-    dst = D_8006F42C;
-    spriteBanks = D_80170720;
-    src = spriteBanks;
-    for (i = 0; i < 0x20; i++) {
+    dst = g_Clut + 0x1430;
```
Comment (sozud) [inline src/servant/tt_000/10E8.c:489]: Is &g_Clut[0xA18] matching?

**Review by sozud — APPROVED** [review]
Nice work

File: src/servant/tt_000/10E8.c (hunk 2 above)
Comment (Xeeynamo) [reply src/servant/tt_000/10E8.c:489]: Yeah, `dst = &g_Clut[0x1430];` and similar are matching too. I'll be happier to change it if it makes it clearer!

**Pull request merged**